src/esp/assets/PTexMeshData.cpp

```diff
@@ -602,8 +602,13 @@ void PTexMeshData::uploadBuffersToGPU(bool forceReload) {
                                   currentMesh->abo);
     currentMesh->abo.setData(adjFaces[iMesh],
                              Magnum::GL::BufferUsage::StaticDraw);
-    currentMesh->mesh.setPrimitive(Magnum::GL::MeshPrimitive::LinesAdjacency)
-        .setCount(currentMesh->ibo.size() / 2)
+    currentMesh->mesh
+        .setPrimitive(Magnum::GL::MeshPrimitive::LinesAdjacency)
+        // Warning:
+        // CANNOT use currentMesh.ibo.size() when calling
+        // setCount because that returns the number of bytes of the buffer, NOT
+        // the index counts
+        .setCount(submeshes_[iMesh].ibo.size())
```

Comment:
Maybe `currentMesh->ibo.size() / sizeof(currentMesh->ibo[0])` then?

Reply:
@erikwijmans : The naming is an issue. See the discussions about this piece of code if you are interested in #132.
A following PR will be made very soon to rename them.